packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -196,8 +196,9 @@ class _TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
   }
 }
 
-class _TooltipPositionDelegate extends SingleChildLayoutDelegate {
-  _TooltipPositionDelegate({
+/// A delegate that positions a tooltip relative to a target.
```

Comment:
if you want to make this public you're going to have to fill in all the documentation on this class :-)